app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -332,7 +331,7 @@ public static boolean checkSpeechAlwaysPref() {
      * @return the boolean
      */
     public boolean checkMicInput() {
-        return micCheck = MediaUtil.isAvailableForVoiceInput(MainActivity.this);
+        return micCheck = MediaUtil.INSTANCE.isAvailableForVoiceInput(MainActivity.this);
```

Comment:
Avoid changes in the main activity. @chiragw15 is working on mainactivity. It may cause conflict.